thetis/coordsys.py

```diff
@@ -0,0 +1,97 @@
+"""
+Generic methods for converting data between different spatial coordinate systems.
+Uses pyproj library.
+"""
+
+import pyproj
+import numpy as np
+
+# NOTE this adds pyproj as dependencency
+# TODO generalize; define unique names for commonly-used systems
+
+
+UTM_ZONE10 = pyproj.Proj(
+    proj='utm',
+    zone=10,
+    datum='NAD83',
+    units='m',
+    errcheck=True)
+SPCS_N_OR = pyproj.Proj(init='nad27:3601', errcheck=True)
+LL_WGS84 = pyproj.Proj(proj='latlong', datum='WGS84', errcheck=True)
+LL_WO = pyproj.Proj(proj='latlong', nadgrids='WO', errcheck=True)  # HARN
+
+
+def convertCoords(x, y, fromSys, toSys):
```

Comment:
I prefer lower case for function names - same for arguments.

More generally: we expect users to use pyproj (to define the fromSys and toSys arguments for starters), so they'll be used to pyproj.transform. If we think this function is useful for general use, we should probably keep it similar to pyproj.transform and have the same argument order - and in the __doc__ say that this is just a "thin wrapper" around pyproj.transform that deals correctly with nans.

Reply:
fixed